interfaces/HO-Portal/src/app/services/jwt.service.ts

```diff
@@ -5,19 +5,21 @@ import { JwtHelperService } from '@auth0/angular-jwt';
   providedIn: 'root',
 })
 export class JwtService {
-  private tokenKey = 'jwt-HO';
+  private tokenKey = 'logged-in-user-HO';
```

Comment:
We might not even need this anymore...
As "Users" and their cookies would be valid for both interfaces at the same time...

But maybe this would need some extra testing...